tests/acceptance/stepDefinitions/notificationsContext.js

```diff
@@ -51,11 +52,25 @@ Then('the notification bell should disappear on the webUI', function () {
   return client.page.phoenixPage().waitForElementNotPresent('@notificationBell')
 })
 
-Then('the user should see {int} notifications on the webUI with these details', function (numberOfNotifications, dataTable) {
-  const dataTableHashed = dataTable.hashes()
-  return client.page.phoenixPage().assertNotificationIsPresent(numberOfNotifications, dataTableHashed)
-})
+Then('the user should see {int} notifications on the webUI with these details',
+  async function (numberOfNotifications, dataTable) {
+    const expectedNotifications = dataTable.hashes()
+    const notifications = await client.page.phoenixPage().getNotifications()
+    assert.strictEqual(
+      notifications.length,
+      numberOfNotifications,
+      'Notification count miss-match!'
+    )
+    for (const element of expectedNotifications) {
+      const userSettings = require('../helpers/userSettings')
```

Comment:
Please keep this require statement outside of the loop, better imported at the top of the page.

Reply:
done